async_remote.md

```diff
@@ -0,0 +1,27 @@
+[other sections soon to follow]
+
+# Knowledge transfer
+
+## Wiki checklist
+
+### Index
+  List and describe all the tools, services and things used in the project.
+  
+### Glossary
+  List and describe most important and required-to-know concepts and terminology used in the project.
+  This section should usually provide newly onboarded team members with important know-how on what means what in the project.
+  Weird acronyms? Overloaded terms which mean different things in normal life? Proper names (_pl. nazwy wlasne_)? They all should be here.
+  
+### Description/guide for specific tools
+  List and describe all developer, design or management tools used in the project.
+  For example, if you're using Zeplin for UI design list it hear and mention what it's for (since not everyone will be familar at first).
```

Comment:
```suggestion
  For example, if you're using Zeplin for UI design list it here and mention what it's for (since not everyone will be familar at first).
```

On the other hand, you say "List and describe alll (...) tools". Isn't there an overlap with the introduction above? CodeClimate appears below.

Reply:
Yeah I think this sections needs to be a bit more developed. 

Using some tools inevitably will be more involved than others (e.g. Zeplin vs Sentry). Those tools will probably require separate articles. For example:

https://github.com/makimo/tomtom-moma-drive-classic/wiki/Firebase-and-Sentry

F and S are briefly mentioned in the Index of the Wiki, but they have a full article as well which goes into detail of how the tools are used and how to use them.